solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,54 @@
+# Some tests in files listed in selective_tests fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# Make 'run-all' ignores the expected fail/error, and test the other tests in these files.
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+TEST = pandas/tests/arrays/string_/test_string.py
+PYTEST_INI_FILE = pytest.ini
+
+OPTS = --max-affinity-cpus=4
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+all: rootfs cpiofs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+	cp $(PYTEST_INI_FILE) appdir/
+
+rootfs: appdir
+	cat selective_tests | while read line; do rm -f appdir/"$$line"; done
+	$(MYST) mkext2 appdir rootfs
+
+rootfs-all: appdir
+	$(MYST) mkext2 appdir rootfs-all
+
+run: rootfs
+	$(MYST_EXEC) rootfs $(OPTS) --app-config-path config.json /usr/local/bin/python3 /app/app.py
+
+run-all: rootfs-all
+	(cat selective_tests | xargs -P 8 $(MYST_EXEC) rootfs-all $(OPTS) --app-config-path config.json \
+		/usr/local/bin/python3 -m pytest -s -q --no-header --skip-slow --skip-db --skip-network 2>&1 > result; \
+		test "$$?" -eq "123")
+	sed -i '$$d' result
```

Comment:
If we maintain the list of failing tests, I don't think we need to do this

Reply:
The problem with pytest was that, I wasn't able to collect the logs with only the test summary above or test names failing. There is no log to easily compare off of. Hence had to edit the verbose run stdio result to just hold the summary and comparing it against what's expected. Also, each file has many tests (approx. 100 each) and there is no way to deselect tests by name... Hence this problem..

Reply:
As far as I have checked, there is no viable way to run just the tests by invoking pytest for each file without modifying the pandas/tests/. The invocation needs to be as a whole of the folder, which is also the expectation from pandas.